### DISCUSSION TITLE
请问一下 青龙面板的脚本什么时候支持lv6用户不投币

### DISCUSSION THREAD

**Discussion by TnZzZHlp** [RayWangQvQ/BiliBiliToolPro#314] [Q&A] [answered]
1

**Comment by RayWangQvQ** [comment]
支持

**Reply by TnZzZHlp** [reply]
好像没有看到有相关的控制变量?默认还是全部用户都投币了,是需要什么特殊变量么

**Comment by RayWangQvQ** [comment, marked as answer]
https://github.com/RayWangQvQ/BiliBiliToolPro/blob/main/docs/configuration.md#3310-lv6%E5%90%8E%E5%BC%80%E5%90%AF%E7%A1%AC%E5%B8%81%E7%99%BD%E5%AB%96%E6%A8%A1%E5%BC%8F

文档里有了

**Reply by TnZzZHlp** [reply]
好的，感谢
